Godeps/_workspace/src/github.com/openshift/openshift-sdn/pkg/cmd/admin/network/make_projects_global.go

```diff
@@ -9,12 +9,12 @@ import (
 	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	kerrors "k8s.io/kubernetes/pkg/util/errors"
 
-	"github.com/openshift/openshift-sdn/plugins/osdn"
-	"github.com/openshift/openshift-sdn/plugins/osdn/ovs"
 	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
 )
 
 const (
+	adminVNID = uint(0)
```

Comment:
Because this change results in all of openshift-sdn being pulled into the
binary, which is the classic "oh I want a constant" -> "oh I just use it
everywhere" -> "oh my source code is now a tangled ball of wax".  Constants
referencing across projects is a terrible idea.

On Tue, Jan 12, 2016 at 10:36 AM, David Eads <notifications@github.com>
wrote:

> In
> Godeps/_workspace/src/github.com/openshift/openshift-sdn/pkg/cmd/admin/network/make_projects_global.go
> <https://github.com/openshift/origin/pull/6496#discussion_r49469185>:
>
> >  	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
> >  )
> >
> >  const (
> > +	adminVNID = uint(0)
>
> I'm not a fan of duplicating constants. Why would we do this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6496/files#r49469185>.
>